packages/showcases/css/stories/components/navigation/tabs/examples/medium.html

```diff
@@ -1,110 +1,174 @@
 <div class="block">
-  <nav aria-label="Tabs list" class="vtmn-tabs vtmn-tabs_size--medium">
-    <ul>
-      <li><a href="" class="selected">Tab Name</a></li>
-      <li><a href="">Tab Name</a></li>
-      <li><a href="">Tab Name</a></li>
-      <li><a href="">Tab Name</a></li>
-    </ul>
-    <div class="vtmn-divider" role="separator"></div>
-  </nav>
+  <ul
+    class="vtmn-tabs vtmn-tabs_size--medium"
+    role="tablist"
+    aria-label="Medium tabs navigation"
+  >
+    <li>
+      <button role="tab" aria-controls="tabpanel-medium-1" aria-selected="true">
+        Tab Name 1
+      </button>
+    </li>
+    <li>
+      <button role="tab">Tab Name 2</button>
+    </li>
+    <li>
+      <button role="tab">Tab Name 3</button>
+    </li>
+    <li>
+      <button role="tab">Tab Name 4</button>
+    </li>
+  </ul>
+  <div class="vtmn-divider" role="separator"></div>
+  <div class="vtmn-w-full vtmn-p-5 vtmn-bg-background-primary">
+    <section
+      role="tabpanel"
+      id="tabpanel-medium-1"
+      aria-labelledby="tabpanel-medium-1"
+      tabindex="0"
```

Comment:
I'm not sure if this is necessary

Reply:
Yes, the is odd as the `aria-labelledby` id shouldn't be the same element